### PR TITLE
updated apigee instructions

### PR DESCRIPTION
**Pull request by dwnoble**

### CODE REVIEW

File: gke/README.md [hunk 1]
```diff
@@ -143,12 +143,18 @@ Use [Apigee](https://cloud.google.com/apigee) for API key management, throttling
 
 - Open GCP console
 - Navigate to Apigee console
-- Click "Enable" button (wait ~1 hour to complete)
+- Click "Customize your setup" button
+- Follow steps, ensuring that apigee is configured to run in the same region as your GKE cluster (setup takes ~1 hour to complete)
+- Use the apigee console to create a new "Instance" in the same region as your GKE cluster. Ensure logging is enabled for the instance.
+- Use the apigee console to create a new "Environment Group" (navigate to "Environment" section the choose "Environment Group"). Add applicable hostnames to the environment.
```
Comment (chejennifer) [inline gke/README.md:149]: navigate to "Environment" section, then choose ...

File: gke/README.md [hunk 2]
```diff
@@ -143,12 +143,18 @@ Use [Apigee](https://cloud.google.com/apigee) for API key management, throttling
...
+- Use the apigee console to create a new "Environment" for the website named "website-environment" with pricing type "Intermediate". Associate this environment with the above instance.
 - Deploy proxy configuration
 
 ```bash
 ../deploy_proxy.sh <ENV>
 ```
 
+- Update manually update the apigee load balancer to use the `dc-website-cert` certificate
```
Comment (chejennifer) [inline gke/README.md:157]: remove the first update?

**Review by chejennifer — APPROVED** [review]
thanks!

File: gke/README.md (hunk 1 above)
Comment (dwnoble) [reply gke/README.md:149]: fixed

File: gke/README.md (hunk 2 above)
Comment (dwnoble) [reply gke/README.md:157]: fixed

**Pull request merged**